{
  "Header": {
    "Original": "CVE-2022-29869 (cifs-utils through 6.14, with verbose logging, can cause an informatio ...)",
    "Line": 162481,
    "ID": "CVE-2022-29869",
    "Description": "(cifs-utils through 6.14, with verbose logging, can cause an informatio ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-5157-1 DLA-3009-1}",
      "Line": 162482,
      "Type": "xref",
      "Bugs": [
        "DSA-5157-1",
        "DLA-3009-1"
      ]
    },
    {
      "Original": "- cifs-utils 2:6.14-1.1 (bug #1010818)",
      "Line": 162483,
      "Type": "package",
      "Package": "cifs-utils",
      "Kind": "fixed",
      "Version": "2:6.14-1.1",
      "BugNo": 1010818
    },
    {
      "Original": "NOTE: https://bugzilla.samba.org/show_bug.cgi?id=15026",
      "Line": 162484,
      "Type": "NOTE",
      "Description": "https://bugzilla.samba.org/show_bug.cgi?id=15026"
    },
    {
      "Original": "NOTE: https://github.com/piastry/cifs-utils/pull/7",
      "Line": 162485,
      "Type": "NOTE",
      "Description": "https://github.com/piastry/cifs-utils/pull/7"
    },
    {
      "Original": "NOTE: https://git.samba.org/cifs-utils.git/?p=cifs-utils.git;a=commit;h=8acc963a2e7e9d63fe1f2e7f73f5a03f83d9c379 (cifs-utils-6.15)",
      "Line": 162486,
      "Type": "NOTE",
      "Description": "https://git.samba.org/cifs-utils.git/?p=cifs-utils.git;a=commit;h=8acc963a2e7e9d63fe1f2e7f73f5a03f83d9c379 (cifs-utils-6.15)"
    }
  ]
}